README.rst

```diff
@@ -88,3 +102,21 @@ More usage examples
 -------------------
 See the `documentation <https://pytorch-datastream.readthedocs.io/en/latest/>`_
 for more usage examples.
+
+
+Python 3.6 support
+==================
+Release `0.3.2` and later have a build specifically supporting `Python 3.6`.
+These builds can be found under the build name `+python3.6`. (E.g. `pytorch-datastream==0.3.2+python3.6`)
```

Comment:
Noone has to think about this `pytorch-datastream==0.3.2+python3.6`. Pip will handle this automatically

Reply:
aha, did not know, so hard to google your way to this information, always hits other things
do you know how the different builds will render on pypi? will the builds be listed as a version? will the supported python versions be given a build or given the release version?

Reply:
I don't know. I looked at numpy and pytorch and I didn't see any special versions